plugins/git/git.plugin.zsh

```diff
@@ -74,6 +74,7 @@ alias gb='git branch'
 alias gba='git branch -a'
 alias gbd='git branch -d'
 alias gbda='git branch --no-color --merged | command grep -vE "^([+*]|\s*($(git_main_branch)|$(git_develop_branch))\s*$)" | command xargs git branch -d 2>/dev/null'
+alias gbdg='git fetch -p && git branch -vv | grep ": gone]" | awk "{print $1}" | xargs git branch -D'
```

Comment:
Since you're using `awk`, you can save on grep. Also, a good practice to use `command` built-in to make sure `awk` or `xargs` will be executed from `$PATH`, and not from a function or a built-in.
To be super safe `git branch --format="%(upstream:track)"` can be used to identify `gone` branches.
```suggestion
alias gbdg="git fetch --prune && git branch --format '%(if:equals=[gone])%(upstream:track)%(then)GONE %(end)%(refname:short)' | command awk '/GONE/{print $2}' | command xargs git branch -D"
```

Reply:
I was happy to see you recent review on this PR, while I was searching for this exact feature!

IMHO, I find the proposed solution in the PR easier to understand than your suggestion, but there might be behavioural differences that I don't understand. If you mean by 'you can save on grep' that `grep` has a performance hit, then maybe that doesn't matter, because `gbdg` would probably be only invoked manually by human users, not automatically in performance-sensitive situations where every millisecond counts. Please explain if and why you think otherwise, as I might be misunderstanding you.

Reply:
Hello @erikhuizinga,

I don't think there are any significant performance penalties calling grep. It's just me being an optimizator. xD
Your code is overall very fine, just one small thing that might make it do a bad thing is `grep ': gone]'`. If on a rare occasion someone writes something something `: gone]` in the first line of their commit message it will be "detected" as a gone branch. And that's what I tried to address via `git branch --format '%(upstream:track)` template.

Reply:
Hi @antonu17, I just found this PR while searching for this feature in omz - would you mind opening a PR with your safer version?